emcc.py

```diff
@@ -1950,7 +1950,7 @@ def phase_linker_setup(options, state, newargs):
 
   if '_main' in settings.EXPORTED_FUNCTIONS:
     settings.EXPORT_IF_DEFINED.append('__main_argc_argv')
-  elif settings.ASSERTIONS:
+  elif settings.ASSERTIONS and not settings.STANDALONE_WASM:
```

Comment:
This is drive by fix.  I notices that we were generating the "main exists but not exported" warning in standalone mode, where it does not apply (since we export `_start` instead and its not optional in the same way)

Reply:
sgtm but please add a comment on the source explaining that.